vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/iso/MobileDrivingLicenceJwsNamespace.kt

```diff
@@ -11,7 +11,7 @@ import kotlinx.serialization.encodeToString
  */
 @Serializable
 data class MobileDrivingLicenceJwsNamespace(
-    @SerialName(IsoDataModelConstants.NAMESPACE_MDL)
+    @SerialName("org.iso.18013.5.1")
```

Comment:
Why?

Reply:
Because it's the only usage of this string value left now